linux/install_ubuntu2004.sh

```diff
@@ -3,7 +3,7 @@
 set -e -u -x
 
 OMEROVER=${OMEROVER:-latest}
-PGVER=${PGVER:-pg11}
+PGVER=${PGVER:-pg12}
```

Comment:
Why the switch to `pg12` while Debian / CentOS 7 are still at `pg11` by default?

Reply:
pg12 is the default version installed. This is a bug
we do not have instructions on how to install pg11

Reply:
Indeed I just realized that https://github.com/ome/omero-install/runs/5110966381 is effectively installing PostgreSQL 12 under the hood.

Since the default is updated in this PR, can we update the GitHub workflow to skip this combination and match the fact only the default version is supported/tested?